README.md

```diff
@@ -51,6 +51,10 @@ jobs:
       uses: kubernetes-sigs/kubebuilder-release-tools@v0.2
       with:
         github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The above way will no longer any more. 
It is deprecated. So, by adding new github action we need to update the note, ensure that we inform what is deprecated and provide the new solution.

Reply:
Okay.
So, if I understand correctly, we need to remove the existing job yaml in the README and add the new one.